### PR TITLE
Refactor helper functions

### PR DESCRIPTION
**Pull request by lawrence14701-zz**
This pull request condenses the size of the analysis router file by reusing the _display_vue_helper function. Some legacy code also was removed from the show_custom_builds function.

### CODE REVIEW

**Comment by jmchilton** [comment]
@galaxybot test this

File: client/galaxy/scripts/entry/analysis/AnalysisRouter.js [hunk 1]
```diff
@@ -339,24 +312,11 @@ export const getAnalysisRouter = (Galaxy) =>
...
-            const vm = document.createElement("div");
-            this.page.display(vm);
-            new workflowImportInstance().$mount(vm);
+            this._display_vue_helper(WorkflowImport);
         },
 
         show_custom_builds: function () {
-            const historyPanel = this.page.historyPanel.historyView;
-            if (!historyPanel || !historyPanel.model || !historyPanel.model.id) {
-                window.setTimeout(() => {
```
Comment (dannon) [inline client/galaxy/scripts/entry/analysis/AnalysisRouter.js]: Worth noting that this does actually change the behavior here, though I don't recall why we had to have a delay in.

Comment (dannon) [reply client/galaxy/scripts/entry/analysis/AnalysisRouter.js]: Ahh.  You can see the problem this was there to address by navigating directly to `http://galaxy_app/custom_builds`.  Central panel loads before the history, causing an issue.

The workaround that was in place isn't great -- the component itself should probably recheck.  Would you like to address that here so there's not a (admittedly minor) regression?

Comment (lawrence14701-zz) [reply client/galaxy/scripts/entry/analysis/AnalysisRouter.js]: Hey Dannon, I removed that change, for now, I'll address it in a separate PR. Thanks.

**Comment by dannon** [comment]
@lawrence14701 Thanks!

File: client/galaxy/scripts/entry/analysis/AnalysisRouter.js (hunk 1 above)
Comment (dannon) [reply client/galaxy/scripts/entry/analysis/AnalysisRouter.js]: @lawrence14701 Thanks!

**Pull request merged**